web/backend/routes/root.ts

```diff
@@ -29,13 +29,23 @@ export default async function (fastify: FastifyInstance) {
       },
     }
   }, async (request) => {
-    let runtimes = await api.getRuntimes()
+    const runtimes = await api.getRuntimes()
+
+    type SelectableRuntime = typeof runtimes[0] & { selected?: boolean }
```

Comment:
We can import the type directly from import { Runtime } from '@platformatic/control'

```suggestion
    type SelectableRuntime = Runtime & { selected?: boolean }
```